### PR TITLE
Indic cleanup2

### PR DESCRIPTION
**Pull request by behdad**
Here's the promised duplication-removal branch. I'm actually happy with it.

### CODE REVIEW

**Comment by behdad** [comment]
@dscorbett This is ready as well, and addresses all the remaining comments of yours.

**Comment by behdad** [comment]
@dscorbett friendly ping here.

File: src/gen-indic-table.py [hunk 1]
```diff
@@ -82,6 +82,58 @@
...
+  'myanmar' : [
+    'VAbv',
+    'VBlw',
+    'VPre',
+    'VPst',
+
+    'IV',
+    'As',
+    'DB',
+    'GB	',
```
Comment (dscorbett) [inline src/gen-indic-table.py:126]: There’s a stray tab here.

**Review by dscorbett — APPROVED** [review]
There is a typo, but it doesn’t end up breaking anything. I approve.

**Pull request merged**